packages/components/src/template-registry.ts

```diff
@@ -40,6 +46,31 @@ export default interface HdsComponentsRegistry {
   'hds/alert/title': typeof HdsAlertTitleComponent;
   HdsAlertTitle: typeof HdsAlertTitleComponent;
 
+  // App Frame
+  'Hds::AppFrame': typeof HdsAppFrameIndexComponent;
+  'hds/app-frame': typeof HdsAppFrameIndexComponent;
+  HdsAppFrame: typeof HdsAppFrameIndexComponent;
+
+  'Hds::AppFrame::Header': typeof HdsAppFrameHeaderComponent;
+  'hds/app-frame/header': typeof HdsAppFrameHeaderComponent;
```

Comment:
These paths should reflect the current structure; they are being used by the `component` helper (for example: `(component "hds/app-frame/parts/modals")`) so updating them will resolve the errors you might have seen in `packages/components/src/components/hds/app-frame/index.hbs`
```suggestion
  'hds/app-frame/parts/header': typeof HdsAppFrameHeaderComponent;
```